README.md

```diff
@@ -33,11 +33,15 @@ Can you afford more? Passionate about the web and JS?
 
 Same benefits as ``jsfounder`` as well as an invitation to join the governance committee.
 
-Limited to first 50 pledges.
+Limited to first 50 pledges. 
+
+## $10000
+
+What are you, cheap? Prove your dedication to neckbeards and donate a grand. Comes with everything mentioned prior AND a special non-blocking handjob.
 
 # Why?
 Okay, this is kind of a vanity TLD, but we freaking build the web every day...  Shouldn't we have a TLD that 
-JS hackers control?
+JS hackers control? We want a destroy-the-internet-fast-to-type TLD!
```

Comment:
This hyphenation is stupid. If anything it should be separated as "destroy-the-internet, fast-to-type"